scripts/helm.sh

```diff
@@ -34,3 +34,12 @@ clean_cluster() {
     return 0
   fi
 }
+
+# install the kubernetes helm binary.
+install_helm() {
```

Comment:
Ah good point, yea let's go this route for now and have the jenkins job(s) determine `HELM_VERSION`, if needing other than canary.